modules/sdp/SDP.js

```diff
@@ -143,9 +143,14 @@ SDP.prototype.toJingle = function(elem, thecreator) {
         }
 
         let ssrc;
+        const simGroup = SDPUtil.findLine(this.media[i], 'a=ssrc-group:SIM');
         const assrcline = SDPUtil.findLine(this.media[i], 'a=ssrc:');
 
         if (assrcline) {
+            if (browser.isFirefox() && simGroup) {
+                // Use the first ssrc from the SIM group since the order of the SSRCs need to be reversed for Firefox.
+                ssrc = simGroup.substring(17).split(' ')[0];
+            }
             ssrc = assrcline.substring(7).split(' ')[0]; // take the first
```

Comment:
Don't you need an `else` here?  It looks like the second `ssrc` assignment overrides the first.

Reply:
🤦‍♀️ I thought I did that but wonder how the test I ran was showing the correct ssrcs in the signaling ..

Reply:
@JonathanLennox fixed it.